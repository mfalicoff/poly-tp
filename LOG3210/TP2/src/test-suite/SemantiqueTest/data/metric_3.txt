Declaration
num a;
num b;
EndDeclaration

a = - b * 1 ;