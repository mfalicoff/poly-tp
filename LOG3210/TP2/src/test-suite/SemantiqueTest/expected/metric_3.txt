{VAR:2, WHILE:0, IF:0, FUNC:0, OP:2}